tools/tock-test-harness/README.md

```diff
@@ -0,0 +1,8 @@
+# tock-test-harness
+## Description
+
+This is a development repository for Tock OS test runner and harness.
+
+Check out the [site](https://goodoomoodoo.github.io/tock-test-harness/) for more info.
```

Comment:
This is a good reference, but can we merge some of it here to keep the doc and code together?